src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx

```diff
@@ -0,0 +1,1132 @@
+        onCommandClick: this.onExecuteQueryClick,
+        commandButtonLabel: label,
+        ariaLabel: label,
+        hasPopup: false,
+        disabled: !this.executeQueryButton.enabled,
+      });
+    }
+
+    if (this.saveQueryButton.visible) {
+      const label = "Save Query";
+      buttons.push({
+        iconSrc: SaveQueryIcon,
+        iconAlt: label,
+        onCommandClick: this.onSaveQueryClick,
+        commandButtonLabel: label,
+        ariaLabel: label,
+        hasPopup: false,
+        disabled: !this.saveQueryButton.enabled,
+      });
+    }
+
+    return buttons;
+  }
+
+  private _buildCommandBarOptions(): void {
+    this.props.tabsBaseInstance.updateNavbarWithTabsButtons();
+  }
+
+  public onChangeContent(newContent: string): void {
+    this.setState({
+      sqlQueryEditorContent: newContent,
+    });
+    if (this.isPreferredApiMongoDB) {
+      if (newContent.length > 0) {
+        this.executeQueryButton = {
+          enabled: true,
+          visible: true,
+        };
+      } else {
+        this.executeQueryButton = {
+          enabled: false,
+          visible: true,
+        };
+      }
+    }
+
+    useCommandBar.getState().setContextButtons(this.getTabsButtons());
+  }
+
+  public onSelectedContent(selectedContent: string): void {
+    if (selectedContent.trim().length > 0) {
+      this.setState({
+        selectedContent: selectedContent,
+        _executeQueryButtonTitle: "Execute Selection",
+      });
+    } else {
+      this.setState({
+        selectedContent: "",
+        _executeQueryButtonTitle: "Execute Query",
+      });
+    }
+    useCommandBar.getState().setContextButtons(this.getTabsButtons());
+  }
+
+  render(): JSX.Element {
+    useCommandBar.getState().setContextButtons(this.getTabsButtons());
+
+    return (
+      <Fragment>
+        <div className="tab-pane" id={this.props.tabId} role="tabpanel">
+          <div className="tabPaneContentContainer">
+            <SplitterLayout vertical={true} primaryIndex={0} primaryMinSize={100} secondaryMinSize={200}>
+              <Fragment>
+                <div className="queryEditor" style={{ height: "100%" }}>
+                  <EditorReact
+                    language={"sql"}
+                    content={this.state.initialEditorContent}
+                    isReadOnly={false}
+                    ariaLabel={"Editing Query"}
+                    lineNumbers={"on"}
+                    onContentChanged={(newContent: string) => this.onChangeContent(newContent)}
+                    onContentSelected={(selectedContent: string) => this.onSelectedContent(selectedContent)}
+                  />
+                </div>
+              </Fragment>
+              <Fragment>
+                {this.isPreferredApiMongoDB && this.state.sqlQueryEditorContent.length === 0 && (
+                  <div className="mongoQueryHelper">
+                    Start by writing a Mongo query, for example: <strong>{"{'id':'foo'}"}</strong>
```

Comment:
Add space after `<strong>{"{'id':'foo'}"}</strong>`

Reply:
Added the space